src/components/job-common.js

```diff
@@ -23,19 +23,44 @@ export const statusType = {
   },
   failed: {
     id: 'failed', // Must match variable name for collection unpacking.
-    label: () => 'Failed',
+    label: () => 'Failed with error(s)',
     icon: style => icon('warning-standard', { size: iconSize, style: { color: colors.danger(), ...style } })
   },
+  paused: {
+    id: 'paused', // Must match variable name for collection unpacking.
+    label: () => 'Paused',
+    icon: style => icon('pause', { size: iconSize, style: { color: colors.dark(), ...style } })
+  },
   running: {
     id: 'running', // Must match variable name for collection unpacking.
     label: () => 'Running',
     icon: style => icon('sync', { size: iconSize, style: { color: colors.dark(), ...style } })
   },
+  canceled: {
+    id: 'canceled', // Must match variable name for collection unpacking.
+    label: () => 'Canceled',
+    icon: style => icon('warning-standard', { size: iconSize, style: { color: colors.dark(), ...style } })
+  },
   submitted: {
     id: 'submitted', // Must match variable name for collection unpacking.
     label: () => 'Submitted',
     icon: style => icon('clock', { size: iconSize, style: { color: colors.dark(), ...style } })
   },
+  queued: {
+    id: 'queued', // Must match variable name for collection unpacking.
+    label: () => 'Queued',
+    icon: style => icon('clock', { size: iconSize, style: { color: colors.dark(), ...style } })
+  },
+  initializing: {
+    id: 'initializing', // Must match variable name for collection unpacking.
+    label: () => 'Initializing',
+    icon: style => icon('cog', { size: iconSize, style: { color: colors.dark(), ...style } })
+  },
+  canceling: {
+    id: 'canceling', // Must match variable name for collection unpacking.
+    label: () => 'Canceling',
+    icon: style => icon('cog', { size: iconSize, style: { color: colors.dark(), ...style } })
```

Comment:
Could probably also match running, or ask for UX help. I would avoid adding a new icon without consultation but re-using an existing one should be fine.